### DISCUSSION TITLE
Interest for contributing.

### DISCUSSION THREAD

**Discussion by Nyabsi** [xerpi/vita2hos#24] [Q&A] [answered]
I'm interested in contributing to vita2hos project but it seems that the libnx32 toolchain is not currently available to the public so I was wondering if there is ETA when It would be available to the public so people are able to compile this project.

**Comment by TSRBerry** [comment, marked as answer]
Could you elaborate on what do you intent to contribute? I might be able to give you access to our dev environment.

There is currently no ETA for public availability, since xerpi is still waiting for a response from the dkp devs.

**Reply by Nyabsi** [reply]
Hey, thanks for your answer. I was thinking of implementing modules such as SceIofilemgr since they should be relatively easy, but I'm also interested just experimenting with rest of the source code.

**Reply by TSRBerry** [reply]
I'll send you an invite! :D
